cdap-ui/app/cdap/components/LogViewer/Playground.tsx

```diff
@@ -124,45 +99,9 @@ const LogViewerPlayground: React.FC<WithStyles<typeof styles>> = ({ classes }) =
       <br />
 
       <div className={classes.logs}>
-        <div>
-          <Button variant="contained" color="primary" onClick={previous}>
-            Previous
-          </Button>
-        </div>
-
-        <div>
-          <pre>
-            {logs.map((logItem, i) => {
-              return (
-                <div key={`${logItem.offset}-${i}`}>
-                  <span>{logItem.log.timestamp}</span>
-                  <span>{` | `}</span>
-                  <span>{logItem.log.message}</span>
-                </div>
-              );
-            })}
-          </pre>
-        </div>
-
-        <div>
-          <Button variant="contained" color="primary" onClick={next}>
-            Next
-          </Button>
-        </div>
-      </div>
-
-      <div className={classes.floatingButton}>
-        <div>
-          <Button variant="contained" color="primary" onClick={getFirst}>
-            Beginning
-          </Button>
-        </div>
-        <br />
-        <div>
-          <Button variant="contained" color="primary" onClick={getLast}>
-            Last
-          </Button>
-        </div>
+        <If condition={!!dataFetcher}>
```

Comment:
I am curious, would this `!!dataFetcher` be different from doing `dataFetcher`?

Reply:
Yeah, especially with typescript type checking. `dataFetcher` is an instance of a class, while `!!dataFetcher` is a boolean checking for existance